### PR TITLE
Implement_and_fixed_Unit_Tests_for_Policy

### PR DESCRIPTION
**Pull request by monstersem**
Fixed units tests in Policy.

Related: APPLINK-17578

### CODE REVIEW

**Comment by monstersem** [comment]
Please rewiew: @AGaliuzov , @Anatoliy-Leshin , @iAndrew5

**Comment by Kozoriz** [comment]
Why a lot of tests were removed?

File: src/components/policy/test/include/mock_policy_listener.h [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 #include "policy/policy_listener.h"
 #include "rpc_base/rpc_base.h"
-#include "table_struct_ext/types.h"
+#include "policy/src/policy/policy_table/table_struct/types.h"
```
Comment (iAndrew5) [inline src/components/policy/test/include/mock_policy_listener.h]: Bad header path. Path should be "directory/file.h". Use CMakeList.

File: src/components/policy/test/generated_code_test.cc [hunk 2]
```diff
@@ -35,8 +35,8 @@
 
 #include "json/reader.h"
 #include "json/value.h"
-#include "table_struct_ext/enums.h"
-#include "table_struct_ext/types.h"
+#include "policy/src/policy/policy_table/table_struct/enums.h"
```
Comment (iAndrew5) [inline src/components/policy/test/generated_code_test.cc]: Bad header path. Path should be "directory/file.h". Use CMakeList.

File: src/components/policy/test/generated_code_test.cc [hunk 3]
```diff
@@ -35,8 +35,8 @@
...
+#include "policy/src/policy/policy_table/table_struct/types.h"
```
Comment (iAndrew5) [inline src/components/policy/test/generated_code_test.cc]: Bad header path. Path should be "directory/file.h". Use CMakeList.

File: src/components/policy/test/include/mock_pt_ext_representation.h [hunk 4]
```diff
@@ -40,7 +40,7 @@
 
 #include "policy/pt_ext_representation.h"
 #include "rpc_base/rpc_base.h"
-#include "table_struct_ext/types.h"
+#include "policy/src/policy/policy_table/table_struct/types.h"
```
Comment (iAndrew5) [inline src/components/policy/test/include/mock_pt_ext_representation.h]: Bad header path. Path should be "directory/file.h". Use CMakeList.

File: src/components/policy/test/include/mock_pt_representation.h [hunk 5]
```diff
@@ -39,7 +39,7 @@
 
 #include "policy/pt_representation.h"
 #include "rpc_base/rpc_base.h"
-#include "table_struct_ext/types.h"
+#include "policy/src/policy/policy_table/table_struct/types.h"
```
Comment (iAndrew5) [inline src/components/policy/test/include/mock_pt_representation.h]: Bad header path. Path should be "directory/file.h". Use CMakeList.

**Comment by iAndrew5** [comment]
Agree with @Kozoriz . It seems to me that too may tests were removed.

**Comment by monstersem** [comment]
A half of tests were used CacheManager, which is removed from Genivi.

**Comment by AGaliuzov** [comment]
Please renmae the branch according gitflow

**Pull request closed without merging**